framework/src/org/checkerframework/framework/util/MultiGraphQualifierHierarchy.java

```diff
@@ -659,6 +665,15 @@ assert getTopAnnotation(a1) == getTopAnnotation(a2) :
         return null;
     }
 
+    // sees if a particular annotation mirror is a polymorphic qualifier
+    private boolean isPolymorphicQualifier(AnnotationMirror qual) {
+        if (AnnotationUtils.containsSame(polyQualifiers.values(), qual)) {
```

Comment:
Just write:
  return AU.containsSame...

It's much shorter and more readable.